distribution/docs/src/main/resources/_contents/_securing/environment-hardening-contents.adoc

```diff
@@ -18,9 +18,7 @@ It is recommended to apply the following security mitigations to the ${branding}
 
 |JMX
 |tampering, information disclosure, and unauthorized access
-a|* Remove `-Dcom.sun.management.jmxremote` from `<${branding}_HOME>/bin/karaf`. +
-* Disable ${branding}'s JMX management `rmiRegistryPort` and `rmiServerPort` (`1099`, `44444`) by removing these entries from `etc/org.apache.karaf.management.cfg`. +
-* Uninstall the management bundle using the command line console: `uninstall management`. +
+a|* Stop the management feature using the command line console: `feature:stop management`. +
```

Comment:
_JMX Connectivity Considerations_ has been added as a note to section:
_Managing
     15. Running
          15.1. Starting_
Screenshot added to PR description for visual context